server/gorm/client.go

```diff
@@ -301,3 +307,32 @@ func (c *Client) Run(ctx context.Context, q storage.Query) storage.Iterator {
 		return nil
 	}
 }
+
+func (c *Client) GetRecentSpecRevisions(ctx context.Context, offset int32, projectID, apiID, versionID string) storage.Iterator {
+	mylock()
+	defer myunlock()
+
+	op := c.db.Select("specs.*").
```

Comment:
Yes, because the query joins the table with an alias of itself we have duplicates of every column and need. We only want to select the columns from the `specs` table because the `grp` table doesn't have all the columns we want.

The `specs.*` part of this query is part of the SQL specification. It [seems to be](https://jakewheat.github.io/sql-overview/sql-2016-foundation-grammar.html#_7_16_query_specification) called an "all fields reference".